orca-api/src/main/java/com/netflix/spinnaker/orca/api/pipeline/models/StageExecution.java

```diff
@@ -86,6 +86,11 @@
 
   void setContext(@Nonnull Map<String, Object> context);
 
+  @Nonnull
+  Map<String, Object> getOthers();
```

Comment:
This is still a very strangely named method, with no comments on what it's used for.